bzt/modules/reporting.py

```diff
@@ -528,7 +532,7 @@ def __init__(self, engine):
         self.log = engine.log.getChild(self.__class__.__name__)
         self.test_suites = OrderedDict()
         bza_report_info = get_bza_report_info(engine, self.log)
-        self.class_name = bza_report_info[0][1] if bza_report_info else "bzt-" + str(self.__hash__())
```

Comment:
Seems like you broke something by just removing this piece of logic

Reply:
@Merilian , I understand that but I don't understand the logic behind the current implementation and what "bza_report_info[0][1] if bza_report_info "  is aimed for.. maybe you or someone can explain so we can find the proper way to handle this?
What I wanted to add is the ability to customize the classname, if that is specified in the configuration.

Reply:
@bitcoder My suggestion here is to leave previous logic and customize class name only when it needed. With default value and nullcheck or something like this